pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/webconsole_proxy.go

```diff
@@ -102,6 +103,18 @@ func (a *webConsolePublicURLAccessor) getPublicConsoleURL() string {
 	return a.updatePublicConsoleURL()
 }
 
+func getServiceCABundle(configMapLister listersv1.ConfigMapLister) string {
```

Comment:
Should be a method on `serviceCABundleRoundTripper` with no parameters, and move the struct and all of its methods here